mp/importer/batcher.py

```diff
@@ -0,0 +1,51 @@
+# this function is intentionally get empty for now
+# this program takes an iterable and splits in a number of batches using the batch_size argument (from context)
+# result is a list that contains the batches (for now)
+
+def run_in_batches(context, iterable, process_one): 		
+    num = int(len(iterable))                                      
```

Comment:
1. len() always returns an int, you should never need to cast here
2. This code only works on a subset of iterables, i.e. those iterables which are "rewindable" (lists, strings). But I don't think you need that limitation. Try find an algorithm that works without you needing to "count" the number of items beforehand.